release-content/migration-guides/relationship_set_risky.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Relationship method set_risky
+pull_requests: [19601]
+---
+
+The trait `Relationship` received a new method, `set_risky`. It is used to alter the entity ID of the parent of this relationship.
+This is needed to keep [other data you can store in these components](https://docs.rs/bevy/latest/bevy/ecs/relationship/trait.Relationship.html#derive)
+untouched at operations that reassign the parent, for example `EntityCommands::add_related`. Previously this could have caused the
+data to be reset to its default value which may not be what you wanted to happen.
```

Comment:
Unsure but isn't "parent" the incorrect term to use here? In the `set_risky` docs you used "related entity".